web/src/SidebarAccount.tsx

```diff
@@ -0,0 +1,268 @@
+
+  &:hover {
+    fill: ${Color.blueLight};
+  }
+`
+let scaleWithBounce = keyframes`
+    0% {
+        transform: scaleY(0)
+    }
+    80% {
+        transform: scaleY(1.05)
+    }
+    100% {
+        transform: scaleY(1)
+    }
+`
+let SidebarAccountMenu = styled.div`
+  position: absolute;
+  left: 0;
+  right: 0;
+  background-color: ${Color.white};
+  color: ${Color.text};
+  border-radius: ${SizeUnit(0.25)};
+  margin-top: ${SizeUnit(0.25)};
+  margin-left: ${SizeUnit(0.15)};
+  margin-right: ${SizeUnit(0.15)};
+  z-index: ${ZIndex.SidebarMenu};
+  transform: scaleY(0);
+  opacity: 0;
+  transform-origin: top center;
+  transition: transform ${AnimDuration.short} ease,
+    opacity ${AnimDuration.short} linear;
+
+  &.isVisible {
+    transform: scaleY(1);
+    animation: ${scaleWithBounce} 300ms ease-in-out;
+    opacity: 1;
+  }
+`
+let SidebarAccountMenuHeader = styled.header`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: 1px dotted ${Color.grayLight};
+  padding-top: ${SizeUnit(0.25)};
+  padding-bottom: ${SizeUnit(0.25)};
+  padding-left: ${SizeUnit(0.5)};
+  padding-right: ${SizeUnit(0.5)};
+`
+let SidebarAccountMenuLogo = styled(TiltCloudLogoSvg)`
+  fill: ${Color.text};
+`
+let SidebarAccountMenuLearn = styled.a`
+  font-family: ${Font.sansSerif};
+  font-size: ${FontSize.smallest};
+  color: ${Color.grayLight};
+`
+let SidebarAccountMenuContent = styled.div`
+  font-family: ${Font.monospace};
+  font-size: ${FontSize.default};
+  color: ${Color.grayLight};
+  padding: ${SizeUnit(0.5)};
+
+  p + p {
+    margin-top: ${SizeUnit(0.25)};
+  }
+
+  strong {
+    font-weight: bold;
+    color: ${Color.text};
+  }
+  small {
+    font-size: ${FontSize.small};
+  }
+  &.is-signedIn {
+    margin-top: ${SizeUnit(0.3)};
+    text-align: right;
+  }
+  &.is-signedOut {
+  }
+`
+let MenuContentTeamInTiltfile = styled.small`
+  opacity: 0;
+  margin-top: 0;
+  display: block;
+  transition: opacity ${AnimDuration.default} ease;
+  transition-delay: ${AnimDuration.short};
+`
+let MenuContentTeamName = styled.strong`
+  transition: background-color ${AnimDuration.default} ease;
+  transition-delay: ${AnimDuration.short};
+`
+let MenuContentTeam = styled.p`
+  &:hover {
+    ${MenuContentTeamInTiltfile} {
+      opacity: 1;
+    }
+    ${MenuContentTeamName} {
+      background-color: ${Color.offWhite};
+    }
+  }
+`
+let MenuContentSignInLink = styled.p`
+  text-align: center;
+`
+export const MenuContentButtonTiltCloud = styled(ButtonLink)`
+  margin-top: ${SizeUnit(0.3)};
+`
+export const MenuContentButtonSignUp = styled(ButtonInput)`
+  margin-top: ${SizeUnit(0.5)};
+  margin-bottom: ${SizeUnit(0.25)};
+`
+
+type SidebarAccountProps = {
+  tiltCloudUsername: string | null
+  tiltCloudSchemeHost: string
+  tiltCloudTeamID: string | null
+}
+
+function notifyTiltOfRegistration() {
+  let url = `${window.location.protocol}//${window.location.host}/api/user_started_tilt_cloud_registration`
```

Comment:
is there some reason you need to re-build the url yourself here? I would have expected `/api/user_started_tilt_cloud_registration` to be equivalent

Reply:
Nope, this was just a bit of mindless copying. Will fix.